packages/client/package.json

```diff
@@ -2,18 +2,66 @@
   "name": "filecoin.storage",
   "version": "0.0.0",
   "description": "API client for filecoin.storage",
-  "main": "index.js",
-  "scripts": {
-    "test": "say sorry"
-  },
+  "type": "module",
   "license": "(Apache-2.0 AND MIT)",
   "repository": {
     "type": "git",
     "url": "git+ssh://git@github.com/filecoin-storage/filecoin.storage.git"
   },
-  "author": "",
   "bugs": {
     "url": "https://github.com/filecoin-storage/filecoin.storage/issues"
   },
-  "homepage": "https://github.com/filecoin-storage/filecoin.storage#readme"
+  "homepage": "https://github.com/filecoin-storage/filecoin.storage#readme",
+  "scripts": {
+    "test": "npm-run-all -p -r mock:api test:all",
+    "test:all": "run-s test:web test:cjs test:es",
+    "test:web": "API_PORT=1337 playwright-test test/*.spec.js",
+    "test:cjs": "API_PORT=1337 mocha dist/test/*.spec.cjs --exit",
+    "test:es": "API_PORT=1337 hundreds mocha test/*.spec.js --exit",
+    "mock:api": "smoke -p 1337 --hooks test/mocks/hooks.js test/mocks/api",
+    "build": "run-p build:*",
+    "build:ts": "tsc --build",
+    "build:cjs": "rollup --config --silent rollup.config.js",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && npx codecov",
+    "typedoc": "typedoc --entryPoints src  --out ../../docs/client",
+    "prepare": "npm run build"
+  },
+  "dependencies": {
+    "@ipld/car": "^2.1.0",
+    "@web-std/blob": "^2.1.0",
+    "@web-std/fetch": "^2.0.1"
+  },
+  "devDependencies": {
+    "@types/mocha": "8.2.2",
+    "hundreds": "0.0.9",
+    "ipfs-car": "^0.2.4",
+    "mocha": "8.3.2",
+    "multiformats": "^7.0.0",
+    "npm-run-all": "^4.1.5",
+    "nyc": "15.1.0",
+    "playwright-test": "^4.1.0",
+    "rollup": "2.50.1",
+    "rollup-plugin-multi-input": "1.3.1",
+    "smoke": "^3.1.1",
+    "typedoc": "0.20.36",
+    "uvu": "0.5.1"
+  },
+  "types": "./dist/src/lib.d.ts",
+  "module": "./src/lib.js",
+  "main": "./dist/src/lib.cjs",
+  "exports": {
+    ".": {
+      "browser": "./src/lib.js",
+      "require": "./dist/src/lib.cjs",
+      "node": "./src/lib.js"
```

Comment:
Should this be "import"? I've not seen "node" before.

Reply:
hmm good question, this part was copied from nft.storage client, and I have not taken the time to fully understand it yet.